c3web-server/src/main/scala/com/ifunsoftware/c3web/annotation/Annotation.scala

```diff
@@ -0,0 +1,14 @@
+package com.ifunsoftware.c3web.annotation
+import com.ifunsoftware.c3web.models.{ File, Tag }
+/**
+ * Created by admin on 17.02.2016.
```

Comment:
It's minor, but i'd prefer to not to place default `created` info headers to project sources.
Instead, it would be nice to add to all source files (.scala ones) standard license headers. You can see an example in the Boot.scala file, for example.